src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,48 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.MotorController;
+
+public class IntakeSubsystem extends SubsystemBase {
+  /**
+   * Creates a new IntakeSubsystem.
+   */
+  //1 DS - extends/retracts intake, 1 neo - intake ball
+  private final DoubleSolenoid mIntakeDoubleSolenoid;
+  private final MotorController mIntakeController;
+
+  public IntakeSubsystem() {
+    mIntakeDoubleSolenoid = new DoubleSolenoid(Constants.kIntakeDoubleSolenoidForwardChannel, Constants.kIntakeDoubleSolenoidReverseChannel);
+    mIntakeController = new MotorController("Intake", Constants.kIntakeDeviceID);
+  }
+
+  public void setIntakeExtended(boolean isExtended) {
```

Comment:
rename isExtended to setExtended or something that doesn't imply if a check of some kind. isExtended makes me think we are checking if something is correct.